engine/access/swagger/generated/api_blocks.go

```diff
@@ -0,0 +1,23 @@
+/*
+ * Access API
+ *
+ * No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
+ *
+ * API version: 1.0.0
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
+package generated
+
+import (
+	"net/http"
+)
+
+func BlocksGet(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+	w.WriteHeader(http.StatusOK)
```

Comment:
that is an excellent suggestion. Also, this generated code is actually not being used. Instead, the `engine/access/swagger/rest_api.go` file provides the actual API implementation. The generated API implementation didn't provide an easy way to add the actual implementation plus I didn't want to change the generated code in case has to be regenerated.

Reply:
So the implementation is supposed to go directly into the generated code, but you're right that this will make things more annoying if we need to regenerate the code. I think the idea is that the API should be stable anyways, and any breaking changes to it would warrant a new API version, which would imply a new implementation.

I think that eventually, we will want to move all the implementation into these generated files. For now, it's probably fine to implement them in a separate file since we could potentially be regenerating the server code if we make any tweaks to the API spec.

Reply:
@vishalchangrani could we create an issue to track this? Thanks :)

Reply:
yup- https://github.com/dapperlabs/flow-go/issues/5997